docs-2.0/nebula-dashboard-ent/4.cluster-operator/operator/update-config.md

```diff
@@ -1,23 +1,40 @@
 # Update config
 
-On **Update Config** page, you can view and modify the service configuration files.
+On **Update Config** page, you can view and update the service configuration files.
+
+## Precautions
+
+You need to restart the corresponding service in the **Service** page after the configuration modification. For details, see [Service](service.md).
 
 ## Entry
 
 1. At the top navigation bar of the Dashboard Enterprise Edition page, click **Cluster Management**.
 2. On the right side of the target cluster, click **Detail**.
 3. On the left-side navigation bar of the page, click **Operation**->**Update Config**.
 
-## Steps
+## Modify configuration
 
 1. Select the type of service whose configuration you want to modify.
 2. Locate the configuration to be modified and click **Edit** in the **Operation** column.
 3. In the pop-up dialog box, you can modify the **Value** individually. They can also be modified uniformly at the top, and you need to click **Apply To All Services** after modification.
 
-  ![config](https://docs-cdn.nebula-graph.com.cn/figures/ds_config_230109_en.png)
+  ![config](https://docs-cdn.nebula-graph.com.cn/figures/ds_config_230327_en.png)
 
 4. Click **Confirm** after the modification is complete.
 
-!!! danger
+## Add configuration
+
+If you need to adjust a parameter that is not included in the configuration file, you need to add the configuration first.
```

Comment:
Should we add instructions about where can the users find parameters that are not included in the configuration file?